scripts/recovery_sync

```diff
@@ -33,20 +33,28 @@ rsync -a /run/systemd/network/ etc/systemd/network
 rsync -a /lib/systemd/network/ lib/systemd/network
 mkdir -p etc/dropbear
 rsync -a /opt/delphix/server/etc/nginx.default ./opt/delphix/server/etc/
-rsync -a /opt/delphix/server/etc/nginx ./opt/delphix/server/etc/
 
 #
-# Delete the access logging from the nginx recovery environment, since the
-# logs won't persist after reboot anyway.
+# Before first engine startup, this directory doesn't exist. We should allow
+# this command to fail without stopping the sync.
 #
-sed -i '/_log /d' ./opt/delphix/server/etc/nginx/nginx.conf
+if [[ -d /opt/delphix/server/etc/nginx ]]; then
```

Comment:
Do we ultimately end up calling recovery_sync when this directory gets populated with the nginx configuration?

Reply:
When we sync the network config and the nginx settings during initial setup, we will call recovery_sync again.